readthedocs/embed/v3/views.py

```diff
@@ -137,21 +138,11 @@ def _get_content_by_fragment(self, url, fragment, doctool, doctoolversion):
         )
 
     def _find_main_node(self, html):
-        main_node = html.css_first("[role=main]")
+        # TODO: find a way to get access to ``project.addons.options_doctool_root_selector``
+        main_node = html.css_first(AddonsConfig.DEFAULT_ROOT_SELECTOR)
         if main_node:
-            log.debug("Main node found. selector=[role=main]")
             return main_node
 
-        main_node = html.css_first("main")
-        if main_node:
-            log.debug("Main node found. selector=main")
-            return main_node
-
-        first_header = html.body.css_first("h1")
-        if first_header:
-            log.debug("Main node found. selector=h1")
-            return first_header.parent
-
```

Comment:
Seems this part isn't in the new selector? It's not backwards compat, so we should probably keep this here for now?